rna_tools/kinwalker/kinwalker.xml

```diff
@@ -0,0 +1,145 @@
+            <param name="used" type="select" label="Barrier Heuristic" help="(--barrier_heuristic)">
+                <option value="M" selected="true">Morgan-Higgs</option>
+                <option value="S">Limit small stacks</option>
+                <option value="B">Barriers</option>
+                <option value="A">All</option>
+            </param>
+            <when value="M">
+                <param name="grouping" type="select" label="How to treat conflict groups" help="(--grouping)">
+                    <option value="standard" selected="true">Standard</option>
+                    <option value="regroup">Re-group</option>
+                </param>
+                <param name="lookahead" type="integer" value="1" label="Number of basepairs that MorganHiggs forms its subpaths from" help="(--lookahead)"/>
+            </when>
+            <when value="S" />
+            <when value="B">
+                <param name="maxkeep" type="integer" value="1" label="Breadth of breadth first search" help="(--maxkeep)"/>
+            </when>
+            <when value="A" />
+        </conditional>
+        <param name="dangle" type="select" label="Dangle value as in VienneRNA package" help="(--dangle)">
+            <option value="0" selected="true">0</option>
+            <option value="1">1</option>
+            <option value="2">2</option>
+        </param>
+        <param name="noLonelyPairs" type="integer" value="2" label="Value of noLonelyPairs as in ViennaRNA" help="(--noLonelyPairs)"/>
+        
+        <param name="transcribed" type="integer" value="1" label="Number of bases initially transcribed" help="0 means all is transcribed (--transcribed)"/>
+        <param name="transcription_rate" type="float" value="200" label="Number of bases transcribed per second" help="(--transcription_rate)"/>
+        <param name="windowsize" type="integer" value="0" label="Max size of substructures considered for folding events during transcription" help="0= all are considered. (--windowsize)"/>
+    </inputs>
+    <outputs>
+        <data format="fasta" name="mfe_struct" label="MFE structure from ${tool.name} on ${on_string}" />
+        <data format="tabular" name="trajectory" label="Trajectory of ${tool.name} on ${on_string}" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="input_sequence" value="test_sequence.fasta" />
+            <param name="init_structure" value="" />
+            <param name="interrupt" value="" />
+            <param name="used" value="M" />
+            <param name="grouping" value="standard" />
+            <param name="lookahead" value="1" />
+            <param name="dangle" value="0" />
+            <param name="noLonelyPairs" value="2" />
+            <param name="transcribed" value="1" />
+            <param name="transcription_rate" value="200" />
+            <param name="windowsize" value="0" />
+            <output name="mfe_struct" file="mfe_struct.fasta" />
```

Comment:
Can you rename the result files to mfe_struct_result.fasta. This way we can distinguish them later, e.g for example during data library upload.